docs/sources/installation/mac.md

```diff
@@ -1,116 +1,315 @@
+
+		$ boot2docker shellinit
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/ca.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/cert.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/key.pem
+			export DOCKER_HOST=tcp://192.168.59.103:2376
+			export DOCKER_CERT_PATH=/Users/mary/.boot2docker/certs/boot2docker-vm
+			export DOCKER_TLS_VERIFY=1
+	
+	The specific paths and address on your machine will be different.
+
+4. To set the environment variables in your shell do the following:
+
+		$ $(boot2docker shellinit)
+		
+	You can also set them manually by using the `export` commands `boot2docker`
+	returns.
+
+5. Run the `hello-world` container to verify your setup.
+
+		$ docker hello-world		
+
+
+## Basic Boot2Docker Exercises
+
+At this point, you should have `boot2docker` running and the `docker` client
+environment initialized. To verify this, run the following commands:
+
+	$ boot2docker status
+	$ docker version
+			
+Work through this section to try some practical container tasks using `boot2docker` VM.
+
+### Access container ports
+
+1. Start an NGINX container on the DOCKER_HOST.
+
+		$ docker run -d -P --name web nginx
+		
+	Normally, the `docker run` commands starts a container, runs it, and then
+	exits. The `-d` flag keeps the container running in the background
+	after the `docker run` command completes. The `-P` flag publishes exposed ports from the
+	container to your local host; this lets you access them from your Mac.
+		
+2. Display your running container with `docker ps` command
+
+		CONTAINER ID        IMAGE               COMMAND                CREATED             STATUS              PORTS                                           NAMES
+		5fb65ff765e9        nginx:latest        "nginx -g 'daemon of   3 minutes ago       Up 3 minutes        0.0.0.0:49156->443/tcp, 0.0.0.0:49157->80/tcp   web  
+
+	At this point, you can see `nginx` is running as a daemon. 
+
+3. View just the container's ports.
+
+		$ docker port web
+		443/tcp -> 0.0.0.0:49156
+		80/tcp -> 0.0.0.0:49157
+	
+	This tells you that the `web` container's port `80` is mapped to port
+	`49157` on your Docker host.
+
+4. Enter the `https://localhost:49157` address (`localhost` is `0.0.0.0`) in your browser:
+
+	   ![Bad Address](/installation/images/bad_host.png)
+
+	This didn't work. The reason it doesn't work is your `DOCKER_HOST` address is
+	not the localhost address (0.0.0.0) but is instead the address of the
+	`boot2docker` VM.
+
+5. Get the address of the `boot2docker` VM.
+
+		$ boot2docker ip
+		192.168.59.103
+		
+6. Enter the `https://192.168.59.103:49157` address in your browser:
+
+	![Correct Addressing](/installation/images/good_host.png)
+
+	Success!
+
+7. To stop and then remove your running `nginx` container, do the following:
+
+		$ docker stop web
+		$ docker rm web
+		
+### Mount a volume on the container
+
+When you start `boot2docker`, it automatically shares your `/Users` directory
```

Comment:
I want to be clear that it is the `boot2docker` process that causes the share to happen.  At this point, the fact we are referring to part of the large application is pretty clear.